### PR TITLE
Examine tweaks - complete xeno examine, armor information

### PR DESCRIPTION
**Pull request by lKiseki**
## About The Pull Request
- Examine, when used by xenos, will now provide complete information instead of omitting most of it. Xenos will sometimes get minor changes to examine text; though most is just flavor, they do get the usual additional information, such as whether someone is hugged, a synthetic, etc.
- Examine will now include more information about modular armor and headwear, such as attachments and modules (this can be seen by both factions):
imagen
- Gets rid of a silly distance check in unconscious/dead examine text.

## Why It's Good For The Game
Xenos will now be able to examine marines to gain important information such as the gear they have, their armor type, and their armor module. This will in turn allow them to have more awareness and strategize around different armor levels and game changers such as Mimir.

## Changelog
:cl: Lewdcifer
add: Examine, when used by xenos, will now provide complete information instead of omitting most of it.
add: Examine will now include more information about modular armor, such as attachments and modules.
code: Gets rid of a distance check in unconscious/dead examine text.
/:cl:

### CODE REVIEW

**Comment by imsxz** [comment]
based

**Comment by aarrisu** [comment]
real...

**Comment by Aporhtonoma** [comment]
xeno buff of the year

**Comment by bowlofcereal** [comment]
cant we just let marines examine eachother to see their jaeger parts as well this doesnt really feel like it needs to be locked to one side

**Comment by lKiseki** [comment]
> cant we just let marines examine eachother to see their jaeger parts as well this doesnt really feel like it needs to be locked to one side

It is not! Marines can also see jaeger parts too.

File: code/modules/mob/living/carbon/human/examine.dm [hunk 1]
```diff
@@ -448,8 +443,10 @@
 						msg += "[span_warning("[t_He] has blood pooling around [t_his] <b>right boot</b>!")]\n"
 
 	if(chestburst == 2)
-		msg += "[span_warning("<b>[t_He] has a giant hole in [t_his] chest!</b>")]\n"
-
+		if(isxeno(user))
+			msg+= "[span_xenowarning("A larva escaped from [t_him]!")]\n"
```
Comment (ZeWaka) [inline code/modules/mob/living/carbon/human/examine.dm:447]:
```suggestion
			msg += "[span_xenowarning("A larva escaped from [t_him]!")]\n"
```

**Review by Lumipharon — APPROVED** [review]

**Comment by Lumipharon** [comment]
@TiviPlus I unno if you still needed to look at this or not, but seems fine for tm at least0

**Pull request merged**